config/locales/en.yml

```diff
@@ -1090,6 +1090,7 @@ en:
         remove_button_confirmation: Are you sure you want to remove this sample from the project?
         error: The sample does not exist within the project.
         files: Files
+        files_ignored: fasta and fastq files must be compressed.
```

Comment:
We should update the error message to let the user know that the following files cannot be uploaded as they must be compressed

Reply:
Updated in 06db0e0dfb1f9b1805dbdf24cf8a3e043be3975e